google_guest_agent/addresses.go

```diff
@@ -531,6 +531,11 @@ func enableNetworkInterfaces() error {
 	}
 }
 
+func isNetworkManagerRunning() bool {
```

Comment:
this is a one-command function, you can just return this logic inline since it's not used elsewhere and isn't suitable for a unit test.

secondly, this adds a dependency on systemd, and assumes 'systemctl' can be found in the path. are you intentionally trying to rely on runCmd returning an error in the case where that's not true? are you sure we only run networkmanager on systemd systems?

Reply:
Oh. I see. For centos/rhel 6, I think it's sysvinit, there are ways to detect init system without by check distros.